### ISSUE TITLE
Find and write recurring blog posts about Qiskitter work

### ISSUE THREAD

**Issue by ryanfmandelbaum** [qiskit-community/qiskit-advocate-mentorship-program#15]
# Description
As advocates, you know more about the Qiskit community than just about anyone. I'm always looking for cool projects that would be interesting to the broader QIskit community, translated into language that even the newest Qiskitter can understand. (See [this blog post](https://medium.com/qiskit/learn-quantum-computing-with-these-seven-projects-7478d90d125a) for example) Essentially, I want to teach one of you how to be a science writer.

# Mentor/s
Ryan Mandelbaum (@ryanfmandelbaum), Senior Technical Writer, main contributor to [Qiskit Medium](https://medium.com/qiskit).

# Type of participant
The ideal participant is passionate not just about their own work, but about other QIskitters' work - and talking about that work. It should be someone looking to hone their writing and storytelling skills, looking for writing opportunities. I'll be offering my knowledge as a writer and an editor in order to leave you detailed feedback and work on making you a better writer overall.

# Number of participants
1-2 advocates.

# Deliverable
A blog post every other month for the length of the mentorship.

**Comment by HuangJunye** [comment]
@EACMichiels @thetronjohnson Can you comment on this issue so that I can assign you? Please also work with your mentor to refine the project, define scope and deliverables and update the project description in this issue.

**Comment by thetronjohnson** [comment]
@HuangJunye Yes, I'll be meeting with my mentor next week to discuss everything

**Comment by ryanfmandelbaum** [comment]
3/9/21: Met, set expectations, which include:
2-3 blogs before end of internship, one of which is a long-form story of approximately 1000 words
WIll meet with Kiran approximately weekly to do writing lessons and check-ins
assigned for 3/22/21, Kiran to come with Ryan with 3 story ideas based on Qiskit and Quantum Community
